README.md

```diff
@@ -163,6 +163,23 @@ user> (h/with-db [db td/game-of-thrones-db-label]
                       [:RETURN "v.name"]]]]
           (println (vec (d/query db query String)))))
 [Ned Catelyn]
+
```

Comment:
should be ``` here 
at the moment the `Oasis` text is within a code block

Reply:
hi @beoliver  I'm no longer using Arango atm & don't foresee being able to spend more time on this.
The code in [this commit](f4ee62e612f0bc5290fbf48fc3b449bde55ae816) works to connect with Oasis.